twitter_ads/error.py

```diff
@@ -72,17 +72,18 @@ class RateLimit(ClientError):
 
     def __init__(self, response, **kwargs):
         super(RateLimit, self).__init__(response, **kwargs)
-        self._retry_after = response.headers.get('retry-after', None)
-        self._reset_at = response.headers.get('rate_limit_reset', None)
+        account_rate_limit_reset = response.headers.get('x-account-rate-limit-reset')
+        rate_limit_reset = response.headers.get('x-rate-limit-reset')
+
+        if account_rate_limit_reset is not None:
```

Comment:
It looks like this is being handled but might be worth testing with endpoint like /5/targeting_criteria/locations that doesn't have account level rate limit just to be sure it works as expected...

Reply:
Added test results to the PR description. It's working as expected.